pkg/cli/admin/mustgather/mustgather.go

```diff
@@ -316,6 +325,27 @@ func (o *MustGatherOptions) Validate() error {
 func (o *MustGatherOptions) Run() error {
 	var errs []error
 
+	if err := os.MkdirAll(o.DestDir, os.ModePerm); err != nil {
+		// ensure the errors bubble up to BackupGathering method for display
+		errs = []error{err}
+		return err
+	}
+
+	f, err := os.Create(path.Join(o.DestDir, "must-gather.logs"))
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "Unable to write must-gather logs: %v. It is possible the destination directory has not been created yet due to early termination\n", err)
+	} else {
+		o.LogWriter = f
+		o.LogWriterMux.Lock()
+		// gets invoked in Complete step before must-gather.logs is created
+		o.LogWriter.WriteString(fmt.Sprintf("[must-gather      ] OUT Using must-gather plug-in image: %s\n", strings.Join(o.Images, ", ")))
```

Comment:
What if user passes `--all-images` flag?. Are we handling it also?.

Reply:
We do. https://github.com/openshift/oc/blob/master/pkg/cli/admin/mustgather/mustgather.go#L208 populates `o.Images` which includes all images. Changes in https://github.com/openshift/oc/pull/1633 extend `completeImages()` which is invoked before this line.